{"name": "test_loginFailMethod", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <AppiumFramwork.tests.LoginTest.LoginTest testMethod=test_loginFailMethod>\n\n    @pytest.mark.run(order=3)\n    def test_loginFailMethod(self):\n        cl.allureLogs(\"App Opened\")\n        self.gt.clickLoginBotton()\n        self.gt.enterEmail()\n        self.gt.enterPassword2()\n        self.gt.clickOnLoginB()\n>       self.gt.verifyAdminScreen()\n\nLoginTest.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <AppiumFramwork.pages.LoginPage.LoginPageTest object at 0x00000268594DFB80>\n\n    def verifyAdminScreen(self):\n        adminScreen = self.isDisplayed(self._pageTitle,\"text\")\n>       assert adminScreen == True\nE       AssertionError\n\n..\\pages\\LoginPage.py:44: AssertionError"}, "steps": [{"name": "App Opened", "status": "passed", "start": 1653640092446, "stop": 1653640092446}, {"name": "Click on Login Button", "status": "passed", "start": 1653640093176, "stop": 1653640093176}, {"name": "Entered email id", "status": "passed", "start": 1653640094859, "stop": 1653640094859}, {"name": "Entered Password", "status": "passed", "start": 1653640096018, "stop": 1653640096018}, {"name": "Clicked on Login Button in Login Screen", "status": "passed", "start": 1653640096592, "stop": 1653640096592}], "attachments": [{"name": "text", "source": "8f0a043e-0469-47ce-85fd-9ef1b241433b-attachment.png", "type": "image/png"}, {"name": "text", "source": "ac351e73-989e-4e74-9f6e-4dc898a6458d-attachment.png", "type": "image/png"}, {"name": "log", "source": "44381e17-8d0b-491a-8b34-7c6dac1c4af5-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "f3f95f40-d50d-4a89-85b6-fe066d3642ad-attachment.txt", "type": "text/plain"}], "start": 1653640092445, "stop": 1653640123217, "uuid": "e41b86d2-c8a6-4044-8a02-a7a9f5823ffd", "historyId": "aeb6c84858cb897d59574054d46d09e3", "testCaseId": "1512c76b4b2d60f059541c0f73c45353", "fullName": "TestSuite.LoginTest#test_loginFailMethod", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('beforeClass', 'beforeMethod')"}, {"name": "tag", "value": "run(order=3)"}, {"name": "suite", "value": "TestSuite"}, {"name": "subSuite", "value": "LoginTest"}, {"name": "host", "value": "BTZVBK3"}, {"name": "thread", "value": "7128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestSuite"}]}